### DISCUSSION TITLE
v1.7.7

### DISCUSSION THREAD

**Discussion by aluminum-ice** [aluminum-ice/pwnagotchi#37] [Announcements]
## What's Changed
* **Removed support from the RPiZW... too much work for an old, obsolete board. Spend $15 and get an RPiZ2W instead.**
* fix: make sure to run fonts.init() so preview.py works by @therocco in https://github.com/aluminum-ice/pwnagotchi/pull/31
* feat: Add in support for Waveshare 2.13inb V4 e-ink display by @therocco in https://github.com/aluminum-ice/pwnagotchi/pull/32
* Updated ups_lite.py for version v1.3 by @brammittendorff in https://github.com/aluminum-ice/pwnagotchi/pull/35
* Update GPS plugin to allow connecting to GPSD by @llamasoft in https://github.com/aluminum-ice/pwnagotchi/pull/36

**Full Changelog**: https://github.com/aluminum-ice/pwnagotchi/compare/v1.7.6...v1.7.7

<hr /><em>This discussion was created from the release <a href='https://github.com/aluminum-ice/pwnagotchi/releases/tag/v1.7.7'>v1.7.7</a>.</em>

**Comment by kolesnikilya** [comment]
Hello. Raspberry pi zero 2 goes blind because of the (waiting for bettercap API to be available). can this be solved somehow?

**Reply by wpa-2** [reply]
Just tested 1.7.7 and mine is fine, so im guessing something you have done has broke it. 
Is it straight away or after a while.

**Reply by kolesnikilya** [reply]
I got raspberries from AliExpress. I install the image through balena. Standard config with home Wi-Fi, The poppy address of the phone for connecting via bluetooth is entered.

**Reply by wpa-2** [reply]
Sorry but im not understanding your issue. 
I suspect something you have done has broke something, reflash and start with a basic config file and go form there.
